src/Features/Core/Portable/Snippets/SnippetChange.cs

```diff
@@ -18,29 +18,29 @@ namespace Microsoft.CodeAnalysis.Snippets;
     /// </summary>
     public readonly ImmutableArray<TextChange> TextChanges;
 
-    /// <summary>
-    /// The position that the cursor should end up on
-    /// </summary>
-    public readonly int CursorPosition;
-
     /// <summary>
     /// The items that we will want to rename as well as the ordering
     /// in which to visit those items.
     /// </summary>
     public readonly ImmutableArray<SnippetPlaceholder> Placeholders;
 
+    /// <summary>
+    /// The position that the cursor should end up on
+    /// </summary>
+    public readonly int FinalCaretPosition;
```

Comment:
This is a rename `CursorPosition` -> `FinalCaretPosition`. First, I wanted to emphasize that this position is used after text changes and placeholders, so `Final`. And second, the standard term for roslyn is 'caret', not 'cursor', so made it more consistent with the rest of the codebase